File: c learning/coding program class/whiteboard loop/prime.c
#include <stdio.h>

int main(){
int n;
scanf("%d",&n);
int c=0;
for(int i=1;i<=n;i++){
    if(n%i==0){
        c=c+1;
    }
}
if(c<=2)
printf("it is prime no");
else
printf("it is non prime no");
return 0;
}

File: Coding blocks dsa/Recursion/prime.java
public class prime{
     public static void main(String[] args) {
        prime_sieve(89);
    }
    public static void prime_sieve(int n){
        boolean[]prime=new boolean[n+1];
        prime[0]=true;
        prime[1]=true;
        int c=0;
        for(int i=2;i*i<=prime.length;i++){
            if(prime[i]==false){
                for(int j=2;i*j<prime.length;j++){
                    prime[i*j]=true;
                }
            }
        }
        for(int i=0;i<prime.length;i++){
            if(prime[i]==false)
            c++;
        }
        System.out.println(c);
    }
}